cmd/kogito/command/test/common.go

```diff
@@ -18,10 +18,12 @@ import (
 	"bytes"
 	"github.com/spf13/cobra"
 	"strings"
-
+	"path/filepath"
+	
 	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/context"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/client"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/client/meta"
+	"github.com/mitchellh/go-homedir"
```

Comment:
Since we're working on this, can you please change the implementation so we reuse the util package to get the home dir? Please see: https://github.com/kiegroup/kogito-cloud-operator/blob/master/pkg/util/envs.go#L43

I'm not against using this dependency, but would be nice to trim a little bit OR use it in the `util` package instead.

Reply:
updated